cmd/server.go

```diff
@@ -12,27 +12,28 @@ import (
 
 	"github.com/ethereum/go-ethereum/crypto"
 
-	"github.com/chainflag/eth-faucet/internal/chain"
-	"github.com/chainflag/eth-faucet/internal/server"
+	"github.com/LiskHQ/lsk-faucet/internal/chain"
+	"github.com/LiskHQ/lsk-faucet/internal/server"
 )
 
 var (
-	appVersion = "v1.1.0"
-	chainIDMap = map[string]int{"goerli": 5, "sepolia": 11155111}
+	appVersion   = "v1.1.0"
+	chainIDMap   = map[string]int{"sepolia": 4202}
+	tokenAddress = flag.String("token-address", os.Getenv("LSK_TOKEN_ADDRESS"), "Contract address of LSK token")
```

Comment:
Please consider adding the information related to the `LSK_TOKEN_ADDRESS` environment variable in the README.